dist/app.min.css

```diff
@@ -453,7 +453,7 @@ mark,
   text-transform: capitalize; }
 
 .text-muted {
-  color: #8c8c8c; }
+  color: #868e96; }
```

Comment:
I cannot explain why these have been regenerated.  I've not changed anything, just run `npm install`.